src/Prefixer.js

```diff
@@ -59,6 +66,15 @@ export function prefix (value, length) {
 		// justify-content
 		case 4968:
 			return replace(replace(value, /(.+:)(flex-)?(.*)/, WEBKIT + 'box-pack:$3' + MS + 'flex-pack:$3'), /s.+-b[^;]+/, 'justify') + WEBKIT + value + value
+		// justify-self
```

Comment:
TODO: size optimisations, i'll submit a patch to the branch this weekend if i get think of something to reduce.

Reply:
Fixed in https://github.com/thysultan/stylis/pull/285/commits/5f9aaf20db9f18b9110bee4850e472994a212b01